standard/template/sections/clause_5_conventions.adoc

```diff
@@ -1,10 +1,34 @@
 == Conventions
 
-This sections provides details and examples for any conventions used in the document. Examples of conventions are symbols, abbreviations, use of XML schema, or special notes regarding how to read the document.
+This section describes the conventions used throughout this Standard, including identifiers, metadata schemas, and referencing mechanisms relevant to the GeoZarr Unified Data Model.
 
 === Identifiers
-The normative provisions in this standard are denoted by the URI
 
-`http://www.opengis.net/spec/{standard}/{m.n}`
+The normative provisions in this Standard are denoted by the base URI:
+
+`http://www.opengis.net/spec/geozarr/1.0`
+
+All requirements, recommendations, permissions, and conformance tests that appear in this document are assigned relative URIs anchored to this base.
+
+For example:
+
+`http://www.opengis.net/spec/geozarr/1.0/conf/core` — refers to the Core Requirements Class of the GeoZarr Unified Data Model.
+
+=== Data Encoding
+
+This Standard specifies the encoding of geospatial data in the Zarr format. Zarr is a chunked, compressed, binary format for n-dimensional arrays, with support for both Version 2 and Version 3 encodings.
+
+The specification makes extensive use of:
+
+- `zarr.json` metadata documents (Zarr v3)
+- `.zgroup`, `.zattrs`, `.zarray` metadata files (Zarr v2)
```

Comment:
In the spirit of cleanly separating the "layers" of this spec, I think we should avoid getting too explicit about the details of how Zarr actually stores data. We don't need to rewrite the Zarr spec in the GeoZarr spec. As much as possible, we should avoid details about Zarr V2 vs. V3 except where there are changes in the Zarr data model (e.g. with dimension names).

Reply:
I simply mirrored the content from other OGC specifications. :)